README.md

```diff
@@ -58,6 +58,8 @@ Place your video in `data` folder and pass the video path to the script. There a
 ./scripts/train_from_video.sh -v {your_video_name} -s {scale} -f {video_fps}
 ```
 
+Nerf use colmap to get the posture of the camera, so you need to download the colmap from [here](https://github.com/colmap/colmap/releases), and rename the directory to "colmap" and put it in the "external" directory.  
```

Comment:
Is this only true for windows?

Reply:
> Is this only true for windows?
It should be true for any platform. But I only test it on Win10 for now. If you are not certain, please delete this passage.